engine/src/main/java/io/zeebe/engine/processing/streamprocessor/ReadonlyProcessingContext.java

```diff
@@ -49,7 +49,7 @@
   ZeebeState getZeebeState();
 
   /** @return the database context for the current actor */
-  DbContext getDbContext();
+  TransactionContext getDbContext();
```

Comment:
Rename TransactionContext -> DbContext or getDbContext to getTransactionContext.